pkgs/shelf_web_socket/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 2.0.0-wip
+## 2.0.0
 
 * Require Dart `^3.0.0`.
 * **BREAKING:**: Remove support for hijacking WebSocket requests that are not
```

Comment:
And this is good because we can delete a bunch of duplicate code, right?

Reply:
The code is already deleted: https://github.com/dart-lang/web_socket_channel/pull/341

Reply:
but this work aligns with that work...hence why we're not as flexible?

Reply:
Yes. `package:web_socket_channel` has a partial copy of the `dart:io` WebSocket implementation that https://github.com/dart-lang/web_socket_channel/pull/341 removes. The code was the cause of the [recent security bug](https://github.com/dart-lang/web_socket_channel/issues/334).